package.json

```diff
@@ -115,12 +118,8 @@
     "ejs": "^3.0.2",
     "eslint": "file:.",
     "eslint-config-eslint": "file:packages/eslint-config-eslint",
-    "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-eslint-plugin": "^5.2.1",
     "eslint-plugin-internal-rules": "file:tools/internal-rules",
-    "eslint-plugin-jsdoc": "^46.9.0",
-    "eslint-plugin-n": "^16.6.0",
-    "eslint-plugin-unicorn": "^49.0.0",
```

Comment:
Can we keep `eslint-plugin-*` dependencies and add them to `ignoreDependencies` in the knip config file for now? I just checked `node_modules` after doing `npm install` on this branch, and although they're installed as dependencies of `eslint-config-eslint`, for some reason they ended up in `eslint-config-eslint/node_modules` instead of top `node_modules`, so linting wouldn't work in IDEs that still use eslintrc.

We'll remove them eventually in https://github.com/eslint/eslint/pull/18011.